.github/workflows/pr.yml

```diff
@@ -18,5 +18,7 @@ jobs:
         run: yarn
       - name: yarn lint
         run: yarn lint
+      - name: yarn build
+        run: yarn build
```

Comment:
We should test that the bundle, when compressed, is <1MB.
Want to do it in this PR or leave it for later?

Reply:
Our heads are in the [same place](https://github.com/Agoric/agoric-sdk/pull/8559#issuecomment-1876111342)!

I think a separate PR is best. I also wonder if `agoric run` / `writeCoreProposal` should be responsible for enforcing this (or, at least warning)

Reply:
New issue here: https://github.com/Agoric/dapp-offer-up/issues/36. Can move it to `agoric-sdk` if you agree this should be a responsibility of `agoric run`